### ISSUE TITLE
Any outstanding errata, etc?

### ISSUE THREAD

**Issue by sean-fitzpatrick** [APEXCalculus/APEXCalculusPTX#304]
Question for @APEXCalculus:

Are there any errata or other items needing attention that are not listed in the issues here?
If not, I think I am going to start working on building the PDFs for this year, and get that out of the way early.
(This is something new I'm trying, rather than trying to finalize textbooks and course outlines simultaneously at the end of August.)

I am not going to do anything about adding end-of-chapter review exercises just yet. Perhaps that's something you'd like to try tackling?

Also, you might be interested to know that over the last few weeks I converted our old linear algebra textbook, which is based on your Matrix Algebra, from LaTeX to PreTeXt. Source is [on GitHub](https://github.com/sean-fitzpatrick/Math1410Text) and you can see the book [in HTML here](https://opentext.uleth.ca/Math1410/). and [in PDF here](https://opentext.uleth.ca/PDF/Math1410_ebook.pdf).

**Comment by sean-fitzpatrick** [comment]
If I have time I will continue working on image descriptions for the last few chapters, but that doesn't impact the PDFs.

**Comment by APEXCalculus** [comment]
I don't have any errata anymore. I think I addressed everything last summer
and nothing new cropped up.

To build the .pdfs - is this a lot of manual work in positioning the
graphics correctly once the "raw" latex is generated? If so, I'm willing to
help with that. I never quite got codespaces to work properly to generate
the needed latex. I think I got close, but there were a few things I was
doing wrong, if I remember correctly. If you put the latex in a repo, I can
wiggle images up/down on a page.

**Comment by sean-fitzpatrick** [comment]
Yeah, the one trouble work PreTeXt that I haven't yet worked out is that we don't have a good way to store the positioning of margin items in the source code. 

(I might try to come up with something...)

But I think I have a workflow that isn't too bad: 

1. Produce new .tex files
2. Compare to last year's adjusted .tex files
3. Copy over all the positional adjustments and other tweaks
4. Build the PDF 
5. Scan through the PDF for anything that's been moved out of place by edits over the last year 
6. Adjust, rebuild, go to step 5.

The main reason it's time consuming is that I've created far too many variants of the PDF. (10, in fact!)

For APEX proper, we have one-sided colour PDF (ebook), two-sided colour PDF (in case someone can afford colour printing), and two-sided black and white PDF for print-on-demand, and each of those can be done with or without the QR codes that link to the videos. 

Then I have both colour and print versions of the PDFs for both of our calculus streams...

**Comment by sean-fitzpatrick** [comment]
In any case, I think I'm at the point where I can do this as efficiently as possible (for an inherently inefficient task).

Where there's still work to be done, for someone who has time: 

- chapter review exercises
- writing WeBWorK code for homework problems

The latter would be a good job for a student, or team of students, if we had money to pay them. 

I think I'd like to redo most of the "terms and concepts" questions using the interactive question types we inherited from Runestone. 

But Vilma Mesa just got her NSF grant approved to do exactly that sort of thing, with APEX and a few other books.

**Comment by sean-fitzpatrick** [comment]
I've figured out a way to store the vertical placement in the PreTeXt source, and use the extra XSL stylesheet to apply those placements when the .tex file is written.

But it's quite possible this is a bad idea. I posted my solution on one of the PreTeXt forums, and probably Rob or @Alex-Jordan  will talk me out of it.

I've tested against the first section in the book, and everything seems to work the way I want it to, with no surprises.
(I'm sure there are some edge cases I've missed that will come out if I try this against the whole book.)

I think the main downside is that the templates I'm modifying change upstream every year, so there is some maintenance, but that's a problem I'm already dealing with, on top of having to edit the .tex files every year.

**Comment by APEXCalculus** [comment]
To me this sounds like good news. The process is inherently inefficient,
yes. There may be a better way that we haven't discovered yet. But it is
WAY more inefficient to have to re-enter vertical placement every time you
run the .tex file vs. doing it once and having it saved in PreTeXt. Yes,
some changes to the content will make these vertical placements obsolete,
but one is still far better off with having some work than not having any
at all.

**Comment by sean-fitzpatrick** [comment]
I think this issue can be safely closed.
